lib/integration/sunlight.rb

```diff
@@ -1,61 +1,99 @@
 class Integration::Sunlight
 
   DOMAIN = 'congress.api.sunlightfoundation.com'
-  LEGISLATORS_ENDPOINT = '/legislators'
+  ENDPOINTS = {
+    bills:       '/bills',
+    legislators: '/legislators'
+  }
   RESULTS_KEY = 'results'
   RESULTS_COUNT_KEY = 'count'
   PAGE_KEY = 'page'
   PER_PAGE_KEY = 'per_page'
   MAX_PER_PAGE = 500 # it's actually 50, but a bigger number doesn't hurt
-  RELEVANT_KEYS = [
-    'birthday',
-    'chamber',
-    # 'contact_form',
-    # 'crp_id',
-    'district',
-    'facebook_id',
-    # 'fax',
-    # 'fec_ids',
-    'first_name',
-    'gender',
-    # 'govtrack_id',
-    # 'icpsr_id',
-    'in_office',
-    'last_name',
-    # 'lis_id',
-    'middle_name',
-    'name_suffix',
-    'nickname',
-    # 'oc_email',
-    # 'ocd_id',
-    'office',
-    'party',
-    'phone',
-    'senate_class',
-    'state',
-    'state_rank',
-    'term_end',
-    'term_start',
-    # 'thomas_id',
-    'title',
-    'twitter_id',
-    # 'votesmart_id',
-    # 'website',
-    # 'youtube_id',
-    'bioguide_id'
-  ]
-  FIELD_NAMES = { 'district' => 'district_code', 'state' => 'state_abbrev' }
-
-  def self.get_legislators( bioguide_id:  nil,
-                           state:        nil,
-                           district:     nil,
-                           senate_class: nil,
-                           get_all:      nil )
-
-    url = legislators_url( bioguide_id:  bioguide_id,
-                           state:        state,
-                           district:     district,
-                           senate_class: senate_class )
+  DEFAULT_PARAMS = { apikey: ENV['SUNLIGHT_KEY'], per_page: MAX_PER_PAGE }
+  JSON_PATHS = {
```

Comment:
yeah, I struggled with the name on this one. I ended up with paths because it contains values like `'cosponsors.legislator.bioguide_id'`, which is a path to a nested key. Or is path the wrong word? I don't think it's an attribute, but I'm happy to hear other suggestions